compiler/src/Language/Bond/Codegen/TypeMapping.hs

```diff
@@ -17,7 +17,7 @@ type system of a target programming language.
 module Language.Bond.Codegen.TypeMapping
     ( -- * Mapping context
       MappingContext(..)
-    , TypeMapping
+    , TypeMapping(..)
```

Comment:
It is also worth considering if we shouldn't export some of the helpers that are usually needed to create a `TypeMapping`. In particular I'm thinking about `elementTypeName`, `instanceTypeName`, `annotatedTypeName`, `aliasTypeName`, `declTypeName` and `declQualifiedTypeName`.